scripts/clean_project.sh

```diff
@@ -6,6 +6,7 @@ echo "Cleaning XpringKit Project"
 
 rm -rf *.xcodeproj
 rm -rf XpringKit/generated
+rm -rf XpringKit/generated/Legacy
```

Comment:
This line is a no op, the `-r` in line 8 is a recursive remove so this directory will also be removed.

Reply:
Oh now I get it. lol